frontend/src/static/css/panels.scss

```diff
@@ -0,0 +1,120 @@
+@import 'colors';
+@import 'z-indices';
+
+.panel-padding {
+  padding: 2rem 1.5rem 2rem 2.2rem;
+}
+
+#app-wrapper {
+  display: flex;
+  height: 100vh;
+  left: 0;
+  position: absolute;
+  top: 0;
+  width: 100%;
+  z-index: z-index('app-wrapper');
+
+  #tab-resize-guide {
+    $tabs-width: 50%;
+
+    background-color: mui-color('black');
+    cursor: col-resize;
+    display: none;
+    height: 100%;
+    position: fixed;
+    right: $tabs-width;
+    width: 2px;
+    z-index: z-index('center-divider');
+  }
+
+  #tabs-wrapper {
+    $tabs-width: 50%;
+
+    background: mui-color('white');
+    flex: 0 0 $tabs-width;
+    overflow-y: scroll;
+    position: relative;
+
+    .tab-close {
+      cursor: pointer;
+      position: fixed;
+      top: calc(50vh - 3rem);
+      z-index: z-index('center-divider');
+
+      svg {
+        background-color: mui-color('grey', '300');
+        color: mui-color('black');
+        padding: 3rem .25rem;
+      }
+    }
+
+    .tab-content {
+      display: flex;
+
+      .tab-panes {
+        text-align: left;
+        width: 100%;
+      }
+    }
+
+    #tab-resize {
+      background-color: mui-color('black');
+      cursor: col-resize;
+      height: 100%;
+      position: fixed;
+      width: 13.250px;
+      z-index: z-index('center-divider');
+    }
+  }
+}
+
+.toolbar {
+  cursor: pointer;
+  float: right;
+}
+
+.sorting {
+  align-items: center;
+  display: flex;
+  margin-top: 1.0rem;
+
+  .mui-select {
+    cursor: pointer;
+    padding-right: 15px;
+  }
+
+  label {
+    overflow-y: hidden;
+  }
+
+  & > label {
+    margin-left: 10px;
+  }
+
+  span {
+    margin-left: .25rem;
+  }
+}
+
+.panel-heading {
+  .title {
+    .group-name {
+      cursor: pointer;
+      font-size: 1.5rem;
```

Comment:
perhaps can try to use `.scss mixin` here? instead of putting the same value of `large-font/medium-font etc` here again.

https://sass-lang.com/documentation/at-rules/mixin

Reply:
Changing this for all `large-font/medium-font/mini-font`. Cool?

Reply:
On second thought, maybe can be in another PR. As the summary component also utilises these fonts. Perhaps can create an issue to do this after merging this PR?

Reply:
Okay can!